h/static/scripts/directives/annotation.coffee

```diff
@@ -265,13 +265,12 @@ AnnotationController = [
           $scope.$emit('annotationUpdate')
 
       # Save highlights once logged in.
-      if highlight and this.isHighlight()
+      if this.isHighlight() and highlight
```

Comment:
I think the `and highlight` should go away in this and below and be collapsed into `isHighlight` function. They were separate because it used to be `and highlight` meaning _and the UI is in highlight mode_. Now, there is no highlight mode, so the question is only ever whether this annotation is a highlight, hence only `isHighlight`, which should hide the implementation detail of `$highlight` from the rest of this controller.